### ISSUE TITLE
Not building on Raspberry Pi

### ISSUE THREAD

**Issue by chipit24** [laradock/php-fpm#29]
I have a Raspberry Pi 3 Model B with docker and docker-compose installed:
```
pi@raspberrypi $ docker --version
Docker version 18.09.0, build 4d60db4

pi@raspberrypi $ docker-compose --version
docker-compose version 1.23.2, build 1110ad0
```

In my Laravel app with Laradock set up, if I run `docker-compose build --no-cache php-fpm` in the `laradock` folder, the `php-fpm` service fails to build, throwing the following error:

```
standard_init_linux.go:207: exec user process caused "exec format error"
ERROR: Service 'php-fpm' failed to build: The command '/bin/sh -c set -xe;     apt-get update -yqq &&     pecl channel-update pecl.php.net &&     apt-get install -yqq       apt-utils       libzip-dev zip unzip &&       docker-php-ext-configure zip --with-libzip &&       docker-php-ext-install zip &&       php -m | grep -q 'zip'' returned a non-zero code: 1
```

I get the same error with `PHP_VERSION` set to `7.1` and with all the `PHP_FPM_INSTALL_*` options set to `false`.

Is there a way to get this working on my Pi?

**Comment by gmatkowski** [comment]
Have you fixed your issue? I got the same error ....

**Comment by chipit24** [comment]
@gmatkowski I have not fixed this issue. I ended up creating a much simpler docker setup that I got working on my RPi.

**Comment by gmatkowski** [comment]
> @gmatkowski I have not fixed this issue. I ended up creating a much simpler docker setup that I got working on my RPi.

I got this working @ nicksonyap/phusion-baseimage-armhf:latest image. But for now it's only available for php 7.2

**Comment by bestlong** [comment]
https://github.com/Laraberries
